src/components/inputs/toggle-input/toggle-input.js

```diff
@@ -1,8 +1,100 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { css } from '@emotion/core';
-import ToggleSwitch from './toggle-switch';
+import styled from '@emotion/styled';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
+import vars from '../../../../materials/custom-properties';
+
+const thumbSmallSize = '13px';
+const thumbBigSize = '26px;';
```

Comment:
are we fine with hardcoded values ?

Reply:
The might have been before. The track and thumb sizes are not bit tricky.

Reply:
Yeah, I guess we could also use something derived from 

```
   standardInputHeight: 32px;
```

🤔